testData/mock/stub/sqlalchemy/orm/attributes.py

```
@@ -0,0 +1,21 @@
+from typing import Generic
+
+# Original signature: (unresolved ancestor classes mess with type checking for tests)
+# class InstrumentedAttribute(QueryableAttribute[_T_co]):
+class InstrumentedAttribute(Generic[_T_co]):
```

Ruff (0.15.11)

[error] Undefined name `_T_co`

(F821)